ocw_data_parser/ocw_data_parser.py

```diff
@@ -287,11 +293,21 @@ def extract_foreign_media_locally(self):
             with open(path_to_containing_folder + file_name, "wb") as file:
                 response = get(media["link"])
                 file.write(response.content)
-            update_file_location(self.master_json, path_to_containing_folder + file_name)
+            if self.static_prefix:
+                update_file_location(self.master_json, self.static_prefix + file_name)
+            else:
+                update_file_location(self.master_json, path_to_containing_folder + file_name)
             log.info("Extracted %s", file_name)
         log.info("Done! extracted foreign media to %s", path_to_containing_folder)
         self.export_master_json()
 
+    def generate_static_site(self):
```

Comment:
Can you add a docstring to new functions, the format used is typically:
```
"""
Generate a static HTML site for a course

Args:
    argname1 (type/class): brief description
    argname2 (type/class): brief description  

Returns:
    (type/class): brief description
"""
```
and you can skip `Args:` and `Returns:` if there aren't any, like for this function.  Examples [here](https://github.com/mitodl/open-discussions/blob/master/channels/utils.py)

Reply:
I added a docstring to this function.